client/sidebar/header/UserDropdown.js

```diff
@@ -95,6 +95,17 @@ const UserDropdown = ({ user, onClose }) => {
 
 	const accountBoxItems = useReactiveValue(getItems);
 
+	const existingStatutText = Object.values(userStatus.list).map((elem) => elem.name);
```

Comment:
Typo with naming

Reply:
Hello @sumukhah, what do you mean ?

Reply:
You mean with StatutText ? I just see it thanks :)